AnimatedTextInput/Classes/AnimatedTextInput.swift

```diff
@@ -531,6 +531,7 @@ extension AnimatedTextInput: TextInputDelegate {
 
     open func textInputDidChange(textInput: TextInput) {
         updateCounter()
+        sendActions(for: .editingChanged)
```

Comment:
I don't understand... what is `sendActions(for: )` ? Is this an extension?
Please provide an implementation for this method, otherwise, the project won't compile.

Reply:
Since you're subclassing `UIControl`, it is good practice to send out appropriate `UIControlEvent`s (cfr https://developer.apple.com/documentation/uikit/uicontrol)

Reply:
Thank you @herre :+1: